tools/systemd/README.md

```diff
@@ -0,0 +1,12 @@
+If you wish to start the webKnossos datastore module after booting, you can use the provided service file.
+There are only a few steps required to set it up.
+
+1. Change the paths in the provided service file from `WK_PATH` to the actual directory, where the webKnossos datastore module is located.
+2. Modify the service file's permissions to `640` by using `chmod 640 webknossos-datastore.service`. 
+3. Change the owner and the group of the file to `root` by using `sudo chown root webknossos-datastore.service` and `sudo chgrp root webknossos-datastore.service`.
+4. Move the `webknossos-datastore.service` file to `/etc/systemd/system/`.
+5. Use `sudo systemctl daemon-reload` to update the systemd files.
+6. Enable the service to start after booting by using `sudo systemctl enable webknossos-datastore.service`.
+7. If you wish to directly start the service, use `sudo systemctl start webknossos-datastore.service`.
```

Comment:
Phew, quite a few steps. The detailed readme is helpful but I think a `./install_as_service.sh` could be an option as well? Maybe it could even be interactive and ask for the correct value for WK_PATH (and for the root password).